### PR TITLE
biz.moneyforward.com

### LINKED ISSUE #108655: biz.moneyforward.com
### Issue URL (Ads)
 [https://biz.moneyforward.com/accounting/basic/45685/#i-6](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fbiz.moneyforward.com%2Faccounting%2Fbasic%2F45685%2F%23i-6)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.3.1
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #108655

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -117,6 +117,9 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt #########
 !
+biz.moneyforward.com##.advads-background
+biz.moneyforward.com##.money-popup-sp 
+biz.moneyforward.com##.money-popup-pc
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/popups.txt:122]: The last two rules of money-popup- can be merged into `biz.moneyforward.com##.advads-effect.is-sticky`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

**Comment by Yuki2718** [comment]
Also add `biz.moneyforward.com##ul[class^="post-sns"]` to Social.

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/popups.txt]: fixed

**Comment by SKEIDs** [comment]
>Also add biz.moneyforward.com##ul[class^="post-sns"] to Social.

done

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**